### PR TITLE
Feature/74638

### PR DESCRIPTION
**Pull request by JKrupinski**
[#74638](https://dev.snowdog.pro/issues/74638)

### CODE REVIEW

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 1]
```diff
@@ -1,95 +1,245 @@
...
+                    id="product-name-sku"
+                    value="<?= $escaper->escapeHtmlAttr($block->prepareInputValue('product-name-sku')); ?>"
+                    title="<?= $escaper->escapeHtmlAttr(__('Product name/sku')); ?>"
+                    minlength="<?= (int) $block->getMinInputLength(); ?>"
+                    placeholder="<?= $escaper->escapeHtmlAttr(__('Search by SKU or Product Name')); ?>"
+                    class="input__field my-orders__search-field"
+                />
             </div>
         </div>
         <div class="field field-50 filters-collapse">
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:57]: formatting

File: Magento_OrderHistorySearch/web/js/mixins/filter-mixin.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      )
+    }
+  }
+
+  return function (targetWidget) {
+    $.widget('mage.myOrdersFilter', targetWidget, myOrdersFilterMixin)
+
+    return $.mage.myOrdersFilter
+  }
+})
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/web/js/mixins/filter-mixin.js:31]: can you add semicolons in this file?

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 3]
```diff
@@ -97,116 +98,226 @@
...
+                field-50
+                col-xs-12
+                col-md-6
+            "
+        >
+            <label
+                class="label"
+                for="order-number"
+            >
+                <?= $escaper->escapeHtml(__('Order Number')); ?></label>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:118]: formatting

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 4]
```diff
@@ -97,116 +98,226 @@
...
+            <div class="input">
+                <input
+                    type="text"
+                    name="order-number"
+                    id="order-number"
+                    value="<?= $escaper->escapeHtmlAttr($block->prepareInputValue('order-number')); ?>"
+                    title="<?= $escaper->escapeHtmlAttr(__('Order Number')); ?>"
+                    placeholder="<?= $escaper->escapeHtmlAttr(__('Enter full or partial number...')); ?>"
+                    minlength="<?=(int)$block->getMinInputLength(); ?>"
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:127]: can you add spaces between `(int)`?

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 5]
```diff
@@ -97,116 +98,226 @@
...
+                field-50
+                col-xs-12
+                col-md-6
+            "
+        >
+            <label
+                class="label"
+                for="order-status"
+            >
+                <?= $escaper->escapeHtmlAttr(__('Order Status')); ?></label>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:145]: formatting

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 6]
```diff
@@ -97,116 +98,226 @@
...
-                        name="order-number"
-                        id="order-number"
-                        value="<?= $escaper->escapeHtmlAttr($block->prepareInputValue('order-number')); ?>"
-                        title="<?= $escaper->escapeHtmlAttr(__('Order Number')); ?>"
-                        placeholder="<?= $escaper->escapeHtmlAttr(__('Enter full or partial number...')); ?>"
-                        minlength="<?=(int)$block->getMinInputLength(); ?>"
-                        class="input__field"
-                    />
-                </div>
+                <?=$block->getOrderStatusSelectElementHtml(); ?>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:153]: can you add space after the opening tag and /* @noEscape */?

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 7]
```diff
@@ -97,116 +98,226 @@
...
+                field-50
+                col-xs-12
+                col-md-6
+            "
+        >
+            <label
+                class="label"
+                for="invoice-number"
+            >
+                <?= $escaper->escapeHtml(__('Invoice Number')); ?></label>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:169]: formatting

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 8]
```diff
@@ -97,116 +98,226 @@
...
+                field-50
+                col-xs-12
+                col-md-6
+            "
+        >
+            <label
+                class="label"
+                for="order-date-from"
+            >
+            <?= $escaper->escapeHtml(__('Order Date')); ?></label>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:206]: formatting

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 9]
```diff
@@ -0,0 +1,434 @@
...
+                <?= $escaper->escapeHtml(__('Invoice Number')); ?></label>
+            <div class="input">
+                <input
+                    type="text"
+                    name="invoice-number"
+                    id="invoice-number"
+                    value="<?= $escaper->escapeHtmlAttr($block->prepareInputValue('invoice-number')); ?>"
+                    title="<?= $escaper->escapeHtmlAttr(__('Invoice number')); ?>"
+                    placeholder="<?= $escaper->escapeHtmlAttr(__('Enter full or partial number...')); ?>"
+                    minlength="<?=(int)$block->getMinInputLength(); ?>"
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:178]: can you add spaces between `(int)`?

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 10]
```diff
@@ -0,0 +1,434 @@
...
+                        <span
+                            class="
+                                sub-label
+                                label
+                                my-orders__field-label
+                            "
+                        >
+                            <?= $escaper->escapeHtml(__('From')); ?>:
+                        </span>
+                        <?=$block->getDateElementToHtml(
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:240]: can you add space after the opening tag and /* @NoEscape */?

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 11]
```diff
@@ -0,0 +1,434 @@
...
+            >
+                <?= $escaper->escapeHtml(__('Created By')); ?></label>
+            <div
+                class="
+                    my-orders__select
+                    select
+                    select--native
+                "
+            >
+                <?=$block->getCreatedBySelectElementHtml(); ?>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:304]: can you add space after the opening tag and /* @NoEscape */?

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 12]
```diff
@@ -0,0 +1,434 @@
...
+                            <span
+                                class="
+                                    sub-label
+                                    label
+                                    my-orders__field-label
+                                "
+                            >
+                                <?= $escaper->escapeHtml(__('Max')); ?>:
+                            </span>
+                            <input type="text"
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:380]: formatting

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 13]
```diff
@@ -0,0 +1,434 @@
...
+                field-order-totals
+                field-50
+                col-xs-12
+                col-md-6"
+            >
+            <label
+                class="label"
+                for="order-total-min"
+            >
+                <?= $escaper->escapeHtml(__('Order Total')); ?></label>
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:319]: formatting

File: Magento_OrderHistorySearch/templates/filters.phtml [hunk 14]
```diff
@@ -0,0 +1,434 @@
...
+                        <span
+                            class="
+                                sub-label
+                                label
+                                my-orders__field-label
+                            "
+                        >
+                            <?= $escaper->escapeHtml(__('To')); ?>:
+                        </span>
+                        <?=$block->getDateElementToHtml(
```
Comment (ofrankowska) [inline Magento_OrderHistorySearch/templates/filters.phtml:272]: can you add space after the opening tag and /* @NoEscape */?

**Review by ofrankowska — CHANGES_REQUESTED** [review]

File: Magento_OrderHistorySearch/templates/filters.phtml (hunk 3 above)
Comment (JKrupinski) [reply Magento_OrderHistorySearch/templates/filters.phtml:118]: this was done on purpose, there is extra space when it's formatted different way

File: Magento_OrderHistorySearch/templates/filters.phtml (hunk 5 above)
Comment (JKrupinski) [reply Magento_OrderHistorySearch/templates/filters.phtml:145]: Same as above

**Pull request merged**